ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.kt

```diff
@@ -269,20 +256,6 @@ class HomeScreenFragment :
     bottomSheetBehavior.state = BottomSheetBehavior.STATE_HIDDEN
   }
 
-  private fun showSurveyLoadingDialog() {
-    if (progressDialog == null) {
-      progressDialog = ProgressDialogs.modalSpinner(context, R.string.survey_loading_please_wait)
-      progressDialog?.show()
```

Comment:
Can we also cleanup the class variable?

Reply:
I meant to do this but forgot, thanks for the heads up!